department-of-veterans-affairs/abd-vro
# For non existant claimid return 400 not 500.

## What was the problem?

When MAS end point called with a claim id that does not exist we were returning 500.


Associated tickets or Slack threads:

- [MCP-2372](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]


MAS end point returns 400 if no claim.

## How to test this PR

There is a end to test added. Make sure that passes/

- Set ENV=end2end-test and build the application locally. 
- ./gradlew end2endTest


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
